cypress/tests/components/dashboard/dashboard.spec.ts

```diff
@@ -0,0 +1,21 @@
+describe('admin dashboard', () => {
+  const DATA_FEDERATION = 'Data Federation';
+
+  before(() => {
+    cy.login();
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('admin dashboard should route properly once user clicks on operator tab under data services', function () {
+    cy.contains('Data Services').click();
+    cy.contains(DATA_FEDERATION).click();
+    cy.url().should('include', '/mcgms/cluster');
+  });
```

Comment:
no need to have separate test file for this... this should be first condition of any card related test suite... no `it` required...
we can add it under utils or helper directory and every test for any card will call this...